third-party/qthread/Makefile

```diff
@@ -132,6 +132,11 @@ ifneq (, $(findstring pgi,$(CHPL_MAKE_TARGET_COMPILER)))
 CHPL_QTHREAD_CFG_OPTIONS += --disable-internal-spinlock
 endif
 
+# if we are using the bundled hwloc then we must specify the prefix for its symbols
+ifeq ($(CHPL_MAKE_HWLOC),bundled)
+CHPL_QTHREAD_CFG_OPTIONS += --with-hwloc-symbol-prefix=$(CHPL_HWLOC_PREFIX)
```

Comment:
I'd throw this up next to `CHPL_QTHREAD_CFG_OPTIONS += --with-hwloc=$(HWLOC_INSTALL_DIR)`